github/PullRequest.py

```diff
@@ -835,12 +835,37 @@ def is_merged(self):
         status, headers, data = self._requester.requestJson("GET", self.url + "/merge")
         return status == 204
 
+    def restore_branch(self):
+        """
+        :calls: `POST /repos/:owner/:repo/git/refs <http://developer.github.com/v3/git/refs>`_
```

Comment:
I mean, it sort of does, but it also doesn't. I'd prefer this linked to create_git_ref using :meth: or something similar.